app/javascript/components/Pages/ProductivityBadFollowUp.js

```diff
@@ -5,9 +5,19 @@ const ProductivityBadFollowUp = ({data, setData, saveDataToDb, steps, service})
   const {isLoading, error} = service
   const { bad_follow_comment } = data.response.attributes
   const [comment, setComment] = useState(bad_follow_comment || '');
+  // const handlingOnClickNext = () => {
```

Comment:
you probably forgot to remove this code?

Reply:
My code is temporary, because the page with the transition is not ready yet. So I commented on the correct code for this page and above my added a comment that it is there temporarily